src/sheet.tsx

```diff
@@ -235,7 +208,7 @@ const Sheet = React.forwardRef<HTMLDivElement, SheetProps>(({onClose, children,
     const {onScroll, overlayStyle, ...dragableSheetProps} = useDraggableSheetProps({closeModal});
 
     useSetModalStateEffect();
-    const bodyStyle = useLockBodyScrollStyleElement();
+    useDisableBodyScroll(modalState !== 'closed');
```

Comment:
I saw no issues using this hook instead of the other one and has some benefits:
- If the document has a scrollbar, the scrollbar is kept. This avoids the content to be resized
- It handles the case of multiple components blocking the scroll